<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(150, 150, 150, 255));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>800</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150, 150, 150, 255), stop:1 rgba(10, 10, 10, 255));</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLCDNumber" name="currentTolerance">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(6, 6, 99);
border-color: rgb(145, 145, 145);
color: rgb(0, 200, 0);</string>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="totalTolerance">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(6, 6, 99);
color: rgb(250, 250, 250);</string>
     </property>
     <property name="intValue" stdset="0">
      <number>50000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Current Tolerance</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Total Tolerance</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>140</y>
      <width>161</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 20, 0, 255), stop:1 rgba(120, 100, 80, 255));</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLCDNumber" name="restLife">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>135</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(6, 6, 99);
color: rgb(250, 250, 250);</string>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>135</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Rest Life</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>135</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Rest Bomb</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="restBomb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>465</y>
       <width>135</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(6, 6, 99);
color: rgb(250, 250, 250);</string>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="score">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>110</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(6, 6, 99);
color: rgb(250, 250, 100);</string>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>135</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Score</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
